AndelaOSP/Andela-Socials
# [Feature #158761414] Deactivate event

#### What Does This PR Do?
Add feature for Deactivate event using GraphQL mutation.
#### Description Of Task To Be Completed
- Add test for `deactivateEvent` mutation and `eventsList` query after event is deactivated.
- Add mutation for `deactivateEvent` and filter event-list for active events only.

#### Any Background Context You Want To Provide?
- Deactivating events should be implemented as a `Delete Event` feature to non-admin end users on the front end. It is only implemented as Deactivate event here to portray to the backend that the event is not being deleted completely but only deactivated from the front-end view.

- I have chosen snapshot testing for [these reasons](http://docs.graphene-python.org/en/latest/testing/#snapshot-testing)

#### How can this be manually tested?
- run `tox`
- run the app, deactivate an event, and query for the event list, confirming that the event is no longer a Node returned by the `eventsList` query.

#### What are the related pivotal tracker stories
[#158761414](https://www.pivotaltracker.com/story/show/158761414)
